client/flutter/lib/home_page.dart

```diff
@@ -75,8 +81,14 @@ class HomePage extends StatelessWidget {
                 ListTile(
                   title: Text(S.of(context).homePagePageSliverListShareTheApp),
                   trailing: Icon(Icons.arrow_forward_ios),
-                  onTap: () => Share.share(
-                      'Check out the official COVID-19 Guide App https://www.who.int/covid-19-app'),
+                  onTap: () {
+                    analytics.logShare(
```

Comment:
I belive all these `analytics.` events are asynchronous so you would have to run `await analytics.logShare`, etc. See https://github.com/FirebaseExtended/flutterfire/blob/master/packages/firebase_analytics/firebase_analytics/example/lib/main.dart#L108

Reply:
Yes, these events are asynchronous. But we don't need a return value & only need to log one here. If we had several asynchronous tasks in a particular order, awaiting was necessary. 
Note that you can't use the `await` keyword inside "onTap".
This works just fine but it's surely a better practice to create a seperate async function & log it there. Let me know if I should do that now.

Reply:
The await keyword can be used inside onTap however as you said, we aren't relying on the result so it's not essential.
`onTap: () async {}`

Reply:
Oh yes, here we **_can_** use async-await. I confused it for another scenario where you can't return a Future.

Reply:
Okay, that seems fine. Can you test the app when turning off the Internet to make sure everything still works? (I assume it will just silently fail and send the logs later when there is Internet connectivity?)